pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -24,13 +25,19 @@ stdenv.mkDerivation {
 
   buildInputs = [
     bison flex perl pkgconfig libpcap lua5 openssl libgcrypt gnutls
-    geoip libnl c-ares python libcap glib
+    geoip libnl c-ares python libcap glib zlib
   ] ++ optional withQt qt4
     ++ optional withGtk gtk;
 
   patches = [ ./wireshark-lookup-dumpcap-in-path.patch ];
 
+  makeFlags = [
+    "DESTDIR=$(out)"
+    "PREFIX="
+  ];
+
   configureFlags = "--disable-usr-local --disable-silent-rules --with-ssl"
+    + " --prefix=/"
```

Comment:
I tested wireshark-qt and wireshark-gtk but i'll remove it and see whether it works without the --prefix.
Could very well be a leftover from experimentation. Thanks for the remark. Ill report back in 10 minutes.